x-pack/plugins/apm/server/routes/alerts/rule_types/get_source_fields.ts

```diff
@@ -0,0 +1,59 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { AggregationsTopHitsAggregation } from '@elastic/elasticsearch/lib/api/types';
+import { SERVICE_GROUP_SUPPORTED_FIELDS } from '../../../../common/service_groups';
+
+export interface SourceDoc {
+  [key: string]: string | SourceDoc;
+}
+
+export function getSourceFieldsAgg(
```

Comment:
Shouldn't this mention that it's the service groups fields. Source fields sounds quite generic - but it's not a generic helper afaict.

```suggestion
export function getServiceGroupsFieldsAgg(
```